test/screenshot/infra/types/cbt-externs.js

```diff
@@ -127,6 +128,7 @@
  *   resolution: !CbtSeleniumResolution,
  *   os: !CbtSeleniumOs,
  *   browser: !CbtSeleniumBrowser,
+ *   caps: !Object<string, string>,
```

Comment:
Revert this file?

Reply:
As discussed offline, we'll keep the changes to this file because they're valid regardless of whether we reference the properties or not.